src/templates/test_helpers/lib.rs.hbs

```diff
@@ -0,0 +1,18 @@
+#[macro_export]
+macro_rules! address_of {
+    ($account:ident) => {
+        ink_e2e::account_id(ink_e2e::AccountKeyring::$account)
+    };
+}
+
+#[macro_export]
+macro_rules! balance_of {
+    ($contract_ref:ident, $client:ident, $address:ident, $account:ident) => \{{
```

Comment:
@prxgr4mm3r

Reply:
@pmikolajczyk41 [Handlebars](https://handlebarsjs.com/guide/#what-is-handlebars) parser throws an error if I don't use '\\' before '{{' because it expects `{{value_that_passed_when_the_template_is_processing}}` instead.

Reply:
oh, okay, it's hbs file - my bad; thanks for pointing this out!